merge_requests.go

```diff
@@ -399,6 +399,31 @@ func (s *MergeRequestsService) GetMergeRequestChanges(pid interface{}, mergeRequ
 	return m, resp, err
 }
 
+// GetMergeRequestParticipants gets a list of merge request participants
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/merge_requests.html#get-single-mr-participants
+func (s *MergeRequestsService) GetMergeRequestParticipants(pid interface{}, mergeRequest int, options ...RequestOptionFunc) ([]*BasicUser, *Response, error) {
+	project, err := parseID(pid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("projects/%s/merge_requests/%d/participants", pathEscape(project), mergeRequest)
+
+	req, err := s.client.NewRequest("GET", u, nil, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var p []*BasicUser
```

Comment:
Maybe add an `s` to indicate it are multiple participants?

Reply:
You mean change the variable name to `ps`?
```suggestion
	var ps []*BasicUser
```

Reply:
Yes indeed... Not a biggie, but would be nice.

Reply:
Done [de6dd49](https://github.com/xanzy/go-gitlab/pull/858/commits/de6dd49087b5822a4f01bbd74dddd0bb32a39760)